Cargo.toml

```diff
@@ -11,13 +11,13 @@ keywords = [ "crypto", "bitcoin" ]
 readme = "README.md"
 
 [dependencies]
-bitcoin = { version= "0.23", features=["use-serde"]}
+bitcoin = { git = "https://github.com/rust-bitcoin/rust-bitcoin", features = [ "use-serde" ] }
 rand="0.7"
 rust-crypto = "0.2"
 serde = "1"
 serde_derive = "1"
 
 [dev-dependencies]
-bitcoin = { version= "0.23", features=["use-serde", "bitcoinconsensus"]}
+bitcoin = { git = "https://github.com/rust-bitcoin/rust-bitcoin", features = [ "use-serde", "bitcoinconsensus" ] }
```

Comment:
I'd just wait until rust-bitcoin makes a release, tbh.

Reply:
Sure, I just had to use the lib now so I made this PR not to forget & share my code